plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/data/managers/AbstractTextPanelEditor.java

```diff
@@ -294,6 +299,24 @@ public void primeEditorValue(@NotNull DBRProgressMonitor monitor, @NotNull Style
         }
     }
 
+    @Nullable
+    @Override
+    public Path getExternalFilePath(@NotNull StyledText control) {
+        try {
+            Path file = Files.createTempFile(DBWorkbench.getPlatform()
+                .getTempFolder(new VoidProgressMonitor(), getFileFolderName()), "file", getFileExtension());
```

Comment:
What if we're opening two files simultaneously? I'm not sure it's ok to call temp file just `file`.

Reply:
They assigned unique number, so it will not cause any issues